src/Adapter/Product/CommandHandler/UpdateProductSeoHandler.php

```diff
@@ -111,11 +120,28 @@ private function fillUpdatableProperties(Product $product, UpdateProductSeoComma
         }
 
         $localizedLinkRewrites = $command->getLocalizedLinkRewrites();
+
         if (null !== $localizedLinkRewrites) {
```

Comment:
This block is a bit redundant now, everything could be done in the loop But you would need to remove the first `if/continue` statement I think or the field will never me updated once it's been filled once